Declarations/Contracts/Generated/Envelope.ts

```diff
@@ -48,11 +48,5 @@ import Base = require('./Base');
          */
         public data: Base;
         
-        constructor()
```

Comment:
Requiring this change is concerning, as it's a file generated from our BOND schema and will eventually get overwritten. Is there a better way to accomplish the reordering?